src/enhanced-patchcore/notebooks/visualize_labels.ipynb

```diff
@@ -408,7 +409,7 @@
    "source": [
     "# Visualize masks with anomaly on the border\n",
     "nb_border_pixel = 20  # To adapt: number of pixel to include in the border\n",
-    "for idx, row in df_anomalous.iterrows():\n",
+    "for idx, row in df_anomalous[df_anomalous[\"pass_id\"] == 1].iterrows():\n",
```

Comment:
why pass_id == 1?

Reply:
We have the masks only for the first video pass for each cable.

Reply:
I see. I asked because I thought we were showing bbox for all passes.

Reply:
We do have bounding boxes for all passes but here we are visualizing the masks.

Reply:
Got it!